service_mesh/v2x/ossm-deploy-production.adoc

```diff
@@ -12,3 +12,7 @@ When you are ready to move from a basic installation to production, you must con
 * Test your configuration in a staging environment.
 
 include::modules/ossm-smcp-prod.adoc[leveloffset=+1]
+
+= Links
```

Comment:
Do we use `= Links`? Not sure I've seen it. This shouldn't be `= Additional resources`?

Reply:
fixed